src/BigInteger.php

```diff
@@ -222,6 +222,18 @@ public function subtract(string $value): BigInteger
     }
 
     /**
+     * Checks if the big integr is the prime number probably.
+     *
+     * @return bool
+     */
+    public function isPrimeNumber(): bool
+    {
+        $result = gmp_prob_prime($this->value);
+
+        return $result == 0 ? false:true;
```

Comment:
Two comments:
1. Always use strict checking, it's faster.
2. This logic is weird, checking for boolean to return a boolean. Might as well return the boolean straight away:

```
return $result !== 0;
```